ssdutil/main.py

```diff
@@ -39,7 +39,7 @@ def import_ssd_api(diskdev):
     except ImportError as e:
         log.log_warning("Platform specific SsdUtil module not found. Falling down to the generic implementation")
         try:
-            from sonic_platform_base.sonic_ssd.ssd_generic import SsdUtil
+            from sonic_platform_base.sonic_storage.ssd import SsdUtil
```

Comment:
>from sonic_platform_base.sonic_storage.ssd import SsdUtil [](http://example.com/codeflow?start=12&length=57)

sonic-utilities submodule update failed with error `2024 Jun  5 08:16:58.507275 vlab-01 ERR CCmisApi: Failed to import default SsdUtil. Error: No module named 'sonic_platform_base.sonic_storage'` in `show_techsupport/test_techsupport_no_secret.py` T0 test, I suspect root cause is this PR, could you help check? test run succeed few days ago [https://elastictest.org/scheduler/testplan/6659944148e3302d8ed2fda8](https://elastictest.org/scheduler/testplan/6659944148e3302d8ed2fda8)

Reply:
Resolving after offline conversation with @maipbui -- gist of which is:

[This pipeline](https://dev.azure.com/mssonic/build/_build/results?buildId=564203&view=logs&j=e2a1d004-6151-514d-dd3b-ba225f574bd9&t=a49dab1b-7bc4-52d2-d09e-ad9af43fe32f&l=597) picks the commit just before the one that introduces sonic_storage to sonic-platform-common. 

[This PR](https://github.com/sonic-net/sonic-buildimage/pull/19042) manually advances the sonic-platform-common, sonic-platform-daemons and sonic-utilities to the latest HEAD.

We can close [sonic-buildimage PR18828](https://github.com/sonic-net/sonic-buildimage/pull/18828) because its goal was to advance sonic-utilities submodule, which is now done.